Interop/ONNX/src/main/java/org/tribuo/interop/onnx/MultiLabelTransformer.java

```diff
@@ -0,0 +1,178 @@
+/*
+ * Copyright (c) 2021, Oracle and/or its affiliates. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.tribuo.interop.onnx;
+
+import ai.onnxruntime.OnnxJavaType;
+import ai.onnxruntime.OnnxTensor;
+import ai.onnxruntime.OnnxValue;
+import ai.onnxruntime.OrtException;
+import com.oracle.labs.mlrg.olcut.provenance.ConfiguredObjectProvenance;
+import com.oracle.labs.mlrg.olcut.provenance.impl.ConfiguredObjectProvenanceImpl;
+import org.tribuo.Example;
+import org.tribuo.ImmutableOutputInfo;
+import org.tribuo.Prediction;
+import org.tribuo.classification.Label;
+import org.tribuo.multilabel.MultiLabel;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashMap;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.logging.Logger;
+
+/**
+ * Can convert an {@link OnnxValue} into a {@link Prediction} or a {@link MultiLabel}.
+ * <p>
+ * Accepts a single tensor representing the probabilities of each label in the batch.
+ * <p>
+ * Predictions are thresholded at {@link #THRESHOLD}, probabilities above this are considered to be present in the
+ * output.
+ */
+public class MultiLabelTransformer implements OutputTransformer<MultiLabel> {
+    private static final long serialVersionUID = 1L;
+    private static final Logger logger = Logger.getLogger(MultiLabelTransformer.class.getName());
+
+    /**
+     * The threshold for conversion into a label.
+     */
+    public static final double THRESHOLD = 0.5;
```

Comment:
Should this be something that the user can adjust?

Reply:
Probably. It's fixed at 0.5 in the binary cross entropy loss in LinearSGDModel, but this should be able to load non-probabilistic models too. I think it forces the `generateProbabilities` flag to true so that'll need to change as well.